### ISSUE TITLE
How to use the clip to take RGB-D images?

### ISSUE THREAD

**Issue by Mr-Neko** [salesforce/ULIP#2]
Thanks for your great work! I noticed that the RGB and RGB-D renderred images are both used in the pipeline of the clip. I wonder that how to use a origin clip to deal with the RGB-D images? Looking forward for your response~

**Comment by Tycho-Xue** [comment]
Hi @Mr-Neko, thanks for the kind words. We actually used RGB images and depth maps separately, the depth maps is similar to gray scale image but the pixel value is related to the depth of the 3d object. Hope it will help you!

**Comment by SiyuanHuang95** [comment]
> 

Then, how to fuse these features?

**Comment by Tycho-Xue** [comment]
we didn't fuse RGB images and depth maps. Each time, we will draw an image from the image pool for that object, it can be either RGB image or depth map.

**Comment by Mr-Neko** [comment]
Thanks for your response! By the way, whether the PointMLP, PointNet++ and PointBert are consturcuted with the default config, or you make your own construct?

**Comment by Tycho-Xue** [comment]
Hi, we used their default config, since we don't change their fine-tune code, we need to put the pre-trained weights back to where it was, so it has to be same.

**Comment by Mr-Neko** [comment]
By the way, I am curious about the hyperparameters of the loss ratio between the three contrastive loss, can you provide the α, β, and θ in your loss?

**Comment by Tycho-Xue** [comment]
the image/text CL loss weight is 0, since we freeze the CLIP's encoders; the image/3d and text/3d weights are both 0.5

**Comment by Tycho-Xue** [comment]
Hi @Mr-Neko , our pre-train code is open-sourced.
